dali/ft/filecopy.cpp

```diff
@@ -2763,20 +2759,21 @@ class AzureFileClient : public CApiCopyClient
                     if (fileCopyOp->HasValue() && fileCopyOp->Value().CopyId.HasValue())
                         fileClient->AbortCopy(fileCopyOp->Value().CopyId.Value().c_str());
                     else // not sure that this can ever happen
-                        IERRLOG("Transfer using API .AbortCopy() failed: unexpected response from API");
+                        IERRLOG("AzureFileClient::AbortCopy() failed: CopyId is empty");
                     status = ApiCopyStatus::Aborted;
                 }
                 catch(const Azure::Core::RequestFailedException& e)
                 {
                     IERRLOG("Abort copy operation failed: %s (code %d)", e.ReasonPhrase.c_str(), static_cast<int>(e.StatusCode));
                     status = ApiCopyStatus::Failed;
                 }
-                break;
+                // drop-through to delete any file remnants
```

Comment:
Use // fallthrough or // fall through so that coverity doesn't report a warning.